### ISSUE TITLE
Lower the learning rate gradually no use? 为什么我逐渐降低lr,对最终误差作用不大呢？

### ISSUE THREAD

**Issue by northeastsquare** [ZJULearning/pixel_link#74]
以前训练caffe的时候使用sgd,也用了momentum了，都是逐渐降低lr,每降低一次，loss都会降很多，为什么对pixel-link 使用mementum optimizer，逐渐降低lr 作用不大呢？

**tensorflow MomentumOptimizer:**  
accumulation = momentum * accumulation + gradient
variable -= learning_rate * accumulation

**caffe sgd:**  
Vt+1=μVt−α∇L(Wt)
Wt+1=Wt+Vt+1
